e2e/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
this should be in the root .gitignore; no need to add it here again

Reply:
separate issue?